src/ctia/task/migrate_es_stores.clj

```diff
@@ -130,32 +136,53 @@
      "false"
      bulk-max-size)))
 
+(def optimized-settings
+  {:index
+   {:number_of_replicas 0
+    :refresh_interval -1}})
```

Comment:
If this is an optimisation why would someone would want not to use it and make it configurable?

I think you should add a documentation in some file about the advantages and drawbacks of the two different approaches.